HHS/Head-Start-TTADP
# [Prod] Use Confluence to populate topic guidance, enable editing of the root cause via the RTR, and remove goal type from the UI

## Description of change
- The topic guidance drawer is now populated via an data from the Confluence RSS feed
- FEI root causes can be edited in the RTR if not completed on an activity report
- Goal type has been removed from the UI

## How to test
- Ensure goal type does not display anywhere in the UI (approved report, RTR, goal forms, etc)
- Create a multirecipient report using the FEI goal and approve it. Visit the RTR for one of those recipients and edit the goal's root causes.
- Confirm that the topic guidance on the goal form is displayed correctly.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1687
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1688
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1459

## Checklists

### Every PR


- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
